src/app/static/src/app/components/plots/table/TableDisplay.vue

```diff
@@ -11,22 +11,23 @@
 </template>
 
 <script lang="ts">
-import { ref, defineComponent, computed, PropType } from "vue";
+import {ref, defineComponent, computed, PropType} from "vue";
 import { AgGridVue } from "ag-grid-vue3";
 import { AgGridEvent } from "ag-grid-community";
 import "ag-grid-community/styles/ag-grid.css";
 import "ag-grid-community/styles/ag-theme-alpine.css";
 import { useStore } from "vuex";
 import { RootState } from "../../../root";
-import {formatOutput, getIndicatorMetadata} from "../utils";
-import { FilterSelection, PlotName } from "../../../store/plotSelections/plotSelections";
-import { IndicatorMetadata, FilterOption, TableMetadata } from "../../../generated";
+import { getIndicatorMetadata } from "../utils";
+import { PlotName } from "../../../store/plotSelections/plotSelections";
+import { IndicatorMetadata, TableMetadata } from "../../../generated";
+import {getColumnDefs, TableHeaderDef} from "./utils";
 
 const defaultColDef = {
     // Set the default filter type
     filter: 'agNumberColumnFilter',
     // Floating filter adds the dedicated row for filtering at the bottom
-    floatingFilter: true,
+    floatingFilter: false,
```

Comment:
Chatted with Rachel about this, she thinks turn it off for now to save some vertical height. We can put it back on if people keep missing them

Reply:
yh think this looks slicker without it

Reply:
yh think this looks slicker without it